docs/release-notes/artifacts/pr0166.yaml

```
@@ -0,0 +1,16 @@
+# --- Release notes artifact ----
+
+schema_version: 1
+changes:
+  - title: Render retry configuration in template and remove retry_interval attribute
+    author: Thanhphan1147
+    type: minor
+    description: |
+      The `retry_interval` attribute is removed since there's no way to configure that in HAproxy.
```

```suggestion
      The `retry_interval` attribute has been removed since there's no way to configure that in HAproxy.
```
Nitpick, but reads a bit better to me